spec/forms/candidate_interface/restructured_work_history/work_history_break_form_spec.rb

```diff
@@ -28,8 +28,53 @@
     it { is_expected.to allow_value(okay_text).for(:reason) }
     it { is_expected.not_to allow_value(long_text).for(:reason) }
 
-    include_examples 'validation for a start date', 'restructured_work_history/work_history_break_form', verify_presence: true
-    include_examples 'validation for an end date that cannot be blank', 'restructured_work_history/work_history_break_form'
+    context 'start_date validations' do
+      let(:model) do
+        described_class.new(start_date_day: start_date_day,
+                            start_date_month: start_date_month,
+                            start_date_year: start_date_year)
+      end
+
+      include_examples 'month and year date validations', :start_date, verify_presence: true, future: true
+
+      describe 'when after the end date' do
+        it 'returns a :before error' do
+          model = described_class.new(start_date_month: '5', start_date_year: '2018',
+                                      end_date_month: '5', end_date_year: '2017')
+
+          expect(model).to be_invalid
+
+          expect(model.errors.added?(:start_date, :before)).to eq(true)
```

Comment:
The `before` validation seems fairly generic - any way to add this to the shared examples? Could maybe have a spec that only runs if validating :start_date and both date fields have values. Otherwise we have to remember to write the inline spec if we want to test this behaviour. Case in point - job_form_spec above isn't currently testing this validation.

Reply:
Good point. Let me look more into this, as I think that would be ideal.